EasyProject/src/main/java/org/example/task1_case1/Main.java

```diff
@@ -0,0 +1,138 @@
+package org.example.task1_case1;
+
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        int col = 3, row = 3;
+        char[][] gameArr = new char[col][row];
+        System.out.print("Enter the cells :");
+        String symbols = eneterCells(col, row, gameArr);
+        printMatrix(gameArr, col, row);
+        System.out.print("Enter the coordinates:");
+        fillMatrix(gameArr, symbols);
+        printMatrix(gameArr, col, row);
+        System.out.println(GameStatus.gameStatus(gameArr, col, row));
+
+
+    }
+
+    public static void printMatrix(char[][] matrix, int lines, int columns) {
```

Comment:
please extract all methods below in separate class

Reply:
done